### ISSUE TITLE
Github's front-end 

### ISSUE THREAD

**Issue by mintobit** [mdo/ama#228]
Hi, Mark.
I am working as a back-end developer and it has been a challenge for me to properly structure and organize the front-end part of the application. I decided to invest some time to actually learn javascript  and approaches to make it maintainable, reusable and testable, instead of just blindly writing jQuery spagetti. Besides from reading (books, articles, conference logs), I find it useful to look what approaches other people are using to solve those problems. Yesterday I looked into github's front-end, and I like the way javascript related selectors are prefixed (js-*). I started to investigate how components interact. The code is a bit hard to follow, because it's compressed. Is there any chance I could see original js files?

**Comment by mdo** [comment]
Sorry, but we won't be sharing our source files publicly :).
